### ISSUE TITLE
melit missing definition

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#451]
I think I hear people say, "melit a kim" a lot and it means, "go looking for clams". This is not in the English definition of melit which is just about opening things. The Palauan definition contains 'melit a kim'. Can people confirm and we can add, "Look for clams" to the English definition?

**Comment by Gnalu** [comment]
Confirm.
Melit: titir: telit.

**Comment by Lukes-Imeyuns** [comment]
The definition is good. But when we say "melit a kim" and "melit oreuer" it
means more than just looking for clams but more about shucking them.

Also, another use of the word is more specific for poking a hole or poked
hole in something versus meling or belsibs.

**Comment by johnbent** [comment]
If you see someone walking out to the ocean and you ask them, "Ke mor?",
can they reply, "Ak mo melit a kim"? And, if so, does that mean, "I'm going
to go shuck clams" or does it mean, "I'm going to look for clams?"

Thanks!

John

**Comment by Gnalu** [comment]
Yes as you don’t say I’m going to look for clams. ‘Ak Mo Melit a Kim/oruer’
regardless you will find any or not.

**Comment by Lukes-Imeyuns** [comment]
Yes, as Mar said.

Saying ak mo melit a kim means finding  and shucking them. The shucking
part is the key.

Now, if they need to collect young clams for a farm then they might say a
ki mo melai a kim el mo er a sers. And we know there's no shucking
involved.

J
